vendor/parser-extended/parsetree.mli

```diff
@@ -914,7 +914,8 @@ and module_declaration =
     {
      pmd_name: string option loc;
      pmd_type: module_type;
-     pmd_attributes: attributes;  (** [... [\@\@id1] [\@\@id2]] *)
+     pmd_attributes_start: attributes;  (** [... [\@\@id1] [\@\@id2]] *)
+     pmd_attributes_end: attributes;  (** [... [\@\@id1] [\@\@id2]] *)
```

Comment:
I would like to start using the new field `pmd_ext_attributes` introduced in #2213 and that will be applied to most types (see all the `*_ext_attributes` in the CST) in the long run, even for the cases where there is no extension allowed.